application/play/scala/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Build app using Maven
```

Comment:
Does this Dockerfile use Maven? I only see `sbt` being used here.

Reply:
Fixed